cli/src/ipfs.ts

```diff
@@ -173,9 +165,113 @@ export class IPFS {
       }
     }
 
-    // TODO: manual update walkthrough
+    const choice = await prompts.select<'distributionRate' | 'burnRate' | 'treasuryAddress'>({
+      message: 'What do you want to update?',
+      choices: [
+        {
+          name: 'Distribution Rate',
+          value: 'distributionRate',
+          description:
+            'Update the current percentage of revenue (RUNE) earned by the treasury to be distributed as rewards.',
+        },
+        {
+          name: 'Burn Rate',
+          value: 'burnRate',
+          description:
+            'Update the current percentage of revenue (RUNE) earned by the treasury to be used to buy FOX and then burn it.',
+        },
+        {
+          name: 'Treasury Address',
+          value: 'treasuryAddress',
+          description: 'Update the THORChain treasury address used to determine revenue earned by the DAO.',
+        },
+      ],
+    })
+
+    switch (choice) {
+      case 'distributionRate': {
```

Comment:
should we validate that burn rate + distrib rate <= 1?

maybe not needed...

Reply:
yeah I thought of this as well. we have it here: https://github.com/shapeshift/rFOX/pull/81/files#diff-689c556b957fa9d068c8def2917359b59e403f664d05274540b5957b5a0d4849R250